### ISSUE TITLE
PHP8: Null-safe operator in chain changes line breaking

### ISSUE THREAD

**Issue by Epskampie** [prettier/plugin-php#1900]
**PHP Plugin 0.17.6**

Line breaking across function calls is normally really nice:

**Input:**
```php
$article
  ->getImage()
  ->setCopyrightName($metas['photo_credits_name ?? null)
  ->setCopyrightUrl($metas['photo_credits_url'] ?? null);
```

**Output is the same:**
```php
$article
  ->getImage()
  ->setCopyrightName($metas['photo_credits_name ?? null)
  ->setCopyrightUrl($metas['photo_credits_url'] ?? null);
```
[Playground link](https://loilo.github.io/prettier-php-playground/#N4IgDgTgpgLjCWUIHkwIPYDsDOIBcok8mMA6vACYwAW+AHAAwA0IMAhgEblW14AsLAK7YoAFU648AMzYAbES2zEA5rKgBFQehhR8M+VBZhqYAGpIlWfCDoA6BiBYwIbeLJUBhdAFtvbAAoAEv74zoKGIBwuAMZQAMowAJ5q1mDYEAC0AEyOINAAjoLw0P4uyn56cgogxCIQMKVs5WyVBgC+LFAU8DDoKGjwWJKgbNgwrSIdNZhgguN4IAA8APzGYAA6mJsAJGz18NFqmwAExxkAfMqwAJJ+VwAUAJQnZ+ciMF5giRDwytQwADk2N4oPdtiD2NgANoAcmM2nQAH1otBujBsIjMGxgTCALrHZbLY6YQSyWTPTCnC7vT7fX7-ACqEFkYIho1h8N6yNRPQxgmZeIJRJJZMeAG4QG0gA)

However if I use the [null safe operator](https://stitcher.io/blog/php-8-nullsafe-operator) `?->`, it gets changed to this:

**Input:**
```php
$article
  ?->getImage()
  ->setCopyrightName($metas['photo_credits_name ?? null)
  ->setCopyrightUrl($metas['photo_credits_url'] ?? null);
```

**Output:**
```php
$article->getImage()?->setCopyrightName(
    $metas["photo_credits_naam"] ?? null,
)->setCopyrightUrl($metas["photo_credits_url"] ?? null);
```
[Playground link](https://loilo.github.io/prettier-php-playground/#N4IgDgTgpgLjCWUIHkwIPYDsDOIBcok8mMA6vACYwAW+AHAAwA0IMAhgEblW14AsLAK7YoAFU648AMzYAbES2zEA5rKgBFQehhR8M+VBZhqYAGpIlWfCDoA6BiBYwIbeLJUBhdAFtvbAAoAEv74zoKGIBwuAMZQAMowAJ5q1mDYEAC0AEyOINAAjoLw0P4uyn56cgogxCIQMKVs5WyVBgC+LFAU8DDoKGjwWJKgbNgwrSIdNZhgguN4IAA8APzGYAA6mJsAJGz18NFqmwAExxkAfMqwAJJ+VwAUAJQnx8sXIjBeYIkQ8MrUMAAcmxvFB7ttQexsABtADkxm06AA+tFoN0YNgkZg2CDYQBdV7LY6YQSyWTPTCnd6wL4-P4AgCqEFk4MhozhCN6KLRPUxgmZ+MJxNJ5IA3CA2kA)

**Expected behavior:**

IMHO `?->` should be treated the same as `->`, so the output should be:

```php
$article
  ?->getImage()
  ->setCopyrightName($metas['photo_credits_name ?? null)
  ->setCopyrightUrl($metas['photo_credits_url'] ?? null);
```

**Comment by Epskampie** [comment]
I'll see if I can create a PR soon, but in the meantime I'd like your perspective!

**Comment by alexander-akait** [comment]
Yep, bug, we should improve this, PR welcome

**Comment by czosel** [comment]
Fixed in `v0.18.0` :tada:

**Comment by Epskampie** [comment]
Awesome, nice to see a fresh release!
